docs/user-guide/home/global-modules/README.md

```diff
@@ -30,17 +30,9 @@ You can create a new project here directly by using the  **+ Project** button.
 
 ![OpenProject projects overview in the global modules menu](openproject_global_modules_projects.png)
 
-On the left you will have the following options:
-
 **All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects**.
```

Comment:
The first entry in the menu is no longer called "All projects" but "Active projects"